lumicks/pylake/tests/data/test_generate_timestamps.py

```diff
@@ -0,0 +1,142 @@
+import numpy as np
+import pytest
+from lumicks.pylake.tests.data.mock_confocal import generate_timestamps
```

Comment:
I might move this file into the actual tests folder. I think it's kinda nice to keep tests and mocking stuff separated. On the other hand, yes it is kinda weird to have tests of our data generation mixed in with actual tests ... thoughts?

Also, as a side note, I think our convention for tests (decided somewhere in the middle, so not entirely consistent) is to use absolute imports for `lumicks.pylake` and relative imports for testing stuff. So just `from .mock_confocal import generate_timestamps`

Reply:
👍

Reply:
You are right, it would be nice to keep mocking and testing stuff separate. However, this specific test is really testing the mocking itself. What do you think about renaming the file from `test_generate_timestamps.py` to `test_mock_confocal.py` to make clear, that you test mocking stuff?